include/GafferScene/SceneAlgo.h

```diff
@@ -80,7 +80,7 @@ GAFFERSCENE_API std::unordered_set<FilteredSceneProcessor *> filteredNodes( Filt
 GAFFERSCENE_API void matchingPaths( const Filter *filter, const ScenePlug *scene, IECore::PathMatcher &paths );
 /// As above, but specifying the filter as a plug - typically Filter::outPlug() or
 /// FilteredSceneProcessor::filterPlug() would be passed.
-GAFFERSCENE_API void matchingPaths( const Gaffer::IntPlug *filterPlug, const ScenePlug *scene, IECore::PathMatcher &paths );
+GAFFERSCENE_API void matchingPaths( const GafferScene::FilterPlug *filterPlug, const ScenePlug *scene, IECore::PathMatcher &paths );
```

Comment:
I think we can drop the `GafferScene::` prefix for a little more brevity, since we're in that namespace already.

Reply:
Updated [here](https://github.com/GafferHQ/gaffer/pull/4061/files#diff-e9b0a5cd76e71e8a996ab34eef4ee2fc4e24b3c79467017c6be291439f4e107fR83) and [here](https://github.com/GafferHQ/gaffer/pull/4061/files#diff-e9b0a5cd76e71e8a996ab34eef4ee2fc4e24b3c79467017c6be291439f4e107fR132).